### ISSUE TITLE
FW2.6 not working with iOS 12

### ISSUE THREAD

**Issue by idzdigital** [rainbowcreatures/FlashyWrappers#48]
I am getting this built time error saying...
ERROR:The content cannot be loaded because there was a problem loading an extension: Error: Requested extension com.rainbowcreatures.FWEncoderANE is not supported for Windows-x86.

While my System is of x64 bit
And its not allowing me to create an .ipa

**Comment by vorlovsky** [comment]
So there were no such error while you were building it before iOS 12?
Actually, it looks like now you try to run your app in Win-based simulator, not on a real device. I can advice you check debug/run settings once again.

**Comment by idzdigital** [comment]
Yes it showed me error even before iOS 12.
so i used flashywrapper sdk 2.5 that worked well,
but sdk 2.5 is not working with ios 12.
The app crashes immediately as it starts without throwing any runtime error.

**Comment by idzdigital** [comment]
Yes with flashywrapper sdk 2.6 it showed me the above error even before iOS 12.
so i used flashywrapper sdk 2.5 that worked well till iOS 11,
but now when i am using sdk 2.5 with iOS 12, the app crashes immediately as it starts without giving any runtime error.

**Comment by rainbowcreatures** [comment]
The first error (Windows-x86) seems like some mixup when building for iOS like @vorlovsky mentioned. As for iOS 12, can't say why the crash happened, it might be really that iOS 12 added something which makes FW crash. Maybe try again with FW 2.6, I think it might use iOS 11 SDK's (vs 2.5 which might be too outdated). I'll test FW with iOS 12 as soon as I get a chance.
